dotcom-rendering/src/web/components/GetMatchNav.importable.tsx

```diff
@@ -1,16 +1,29 @@
+import { css } from '@emotion/react';
+import { from } from '@guardian/source-foundations';
 import { useApi } from '../lib/useApi';
 
 import { Placeholder } from './Placeholder';
 
 import { MatchNav } from './MatchNav';
+import { ArticleHeadline } from './ArticleHeadline';
 
 type Props = {
 	matchUrl: string;
+	headlineString: string;
+	format: ArticleFormat;
+	tags: TagType[];
+	webPublicationDateDeprecated: string;
```

Comment:
Completely tangential, and not related to this PR at all, do why this publication date is deprecated and is there another date property we should be using instead? Maybe this is part of the frontend date migration work that never got completed?

Reply:
I think you're right. There was another piece of work around dates which probably involved renaming this. Not really sure why this was.

It does feel a bit odd to be using it in this PR but I don't want to introduce _changes_ here as that feels apart from what I'm doing.